doc/source/config.rst

```diff
@@ -37,6 +37,45 @@ several config files and pick from them using `-c`. If you only wish to control
 the specific tests that are to be run (and not their parameters) then using
 `-s` or `-t` on the command line may be more appropriate.
 
+Also you can configure bandit via
+`pyproject.toml <https://www.python.org/dev/peps/pep-0518/>`_ file. In this
+case you would explicitly specify the path to configuration via `-s` too.
+For example:
+
+.. code-block:: python
+    [tool.bandit.profiles.test_1]
```

Comment:
Previewing this page reveals that this block won't be rendered at all, a space is required between lines 45 and 46.

```suggestion
.. code-block:: python

    [tool.bandit.profiles.test_1]
```